src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -3,3 +3,6 @@ framework:
         storage_id: session.storage.mock_file
     profiler:
         enabled: false
+
+sulu_document_manager:
```

Comment:
This way we are coupling the `PreviewBundle` to the `DocumentManagerBundle`, but I don't know any other solution at the moment...